selfdrive/boardd/boardd.cc

```diff
@@ -302,6 +302,7 @@ std::optional<bool> send_panda_states(PubMaster *pm, const std::vector<Panda *>
   auto pss = evt.initPandaStates(pandas.size());
 
   std::vector<health_t> pandaStates;
+  std::vector<std::array<can_health_t, 3>> pandaCanStates;
```

Comment:
also don't need the outer vector, can replace the health one too

Reply:
confused why... vector of can_heath_t array objects for multiple pandas.

Reply:
I meant you can make it an array too

Reply:
it is unknown size same as pandaStates.

Reply:
it's `pandas.size()`